README.md

```diff
@@ -127,61 +130,18 @@ Advanced users may want to choose a particular Sparking Water / H2O version (spe
 |               | 2.1.2                   | 3.10.4.2    | "rel-ueno"       |        "2"         |
 |               | 2.1.1                   | 3.10.4.2    | "rel-ueno"       |        "2"         |
 |               | 2.1.0                   | 3.10.3.2    | "rel-tverberg"   |        "2"         |
-|               |                         |             |                  |                    |
-| 2.0.*         | 2.0.28                  | 3.18.0.8    | "rel-wolpert"    |        "8"         |
```

Comment:
Can you explain why this is not supported anymore? Can't this information still be useful to people?

Reply:
We don't want people to use these versions. They are not mentioned on download page, pipy, in Sparkling Water documentations. RSparkling is the last step.

If anybody needs to use the older version, they have to make some effort or upgrade